vigetlabs/wordpress-site-starter
# [n/a] Moving to theme colors vars

# Summary

An idea that Brian had about moving the main accent color to a variable to it could be changed in once location and it would roll out for the whole site. 

Instead of creating a new config file, I kept everything within the TW config. 
The two main locations this helps with is the `buttons.js` and the `theme.json` files. With this change, we no longer have to update the color in those files as they are now dynamic. 
The steps to update the color would be to change `accentColor` to whatever color name you need it to be. Then update all the colors under `[accentColor]` and you have restyled the theme. 

The one blocker with this is that the vars don't pull over into CSS without a lot of work which may just make it more confusing to style than it would help. So for that case I changed all the `sky` colors in the CSS to `gray`. When we build out a theme we will need to update a lot of the style anyways.

## Issues

* NA

## Testing Instructions

1. Update the `accentColor` so something. Does not really matter as the JS and  theme.json are pulling in that var but allows you to use that var name in your CSS.
2. Update the colors under `[accentColor]`. 
3. Make sure the colors on the site are updated. 

## Screenshots

| default | custom |
|--------|--------|
| ![wpstarter ddev site_this-is-a-link_sample-page_](https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/353dfe22-7aef-4689-82f2-ce34f1276a8a) | ![wpstarter ddev site_this-is-a-link_sample-page_ (1)](https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/1dbb4ef2-5ed4-4152-ae9a-2f452be93dc4) |
